client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -57,6 +58,9 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = () =>
 				<Route exact path={ Step.DesignSelection }>
 					{ ! siteVertical ? <Redirect to={ Step.IntentGathering } /> : <DesignSelector /> }
 				</Route>
+				<Route exact path={ Step.Signup }>
+					<SignupForm />
```

Comment:
Note for future, related PR: 

Create an action to show modal, so we can flip it open at any point in the journey